modules/installation-aws-marketplace-subscribe.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws/installing-aws-government-region.adoc
+// * installing/installing_aws/installing-aws-user-infra.adoc
+
+ifeval::["{context}" == "installing-aws-customizations"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-aws-government-region"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-aws-user-infra"]
+:upi:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-aws-marketplace-subscribe_{context}"]
+= Obtaining an AWS Marketplace image
+If you are deploying an {product-title} cluster using an AWS Marketplace image, you must first subscribe through AWS. Subscribing to the offer provides you with the AMI id that the installation program uses to deploy worker nodes.
+
+.Prerequisites
+
+* You have an AWS account to purchase the offer. This account does not have to be the same account that is used to install the cluster.
+
+.Procedure
+
+. Complete the {product-title} subscription from the link:https://aws.amazon.com/marketplace/fulfillment?productId=59ead7de-2540-4653-a8b0-fa7926d5c845[AWS Marketplace].
+ifdef::ipi[]
+. Record the AMI id for your specific region. As part of the installation process, you must update the `install-config.yaml` file with this value before deploying the cluster.
+endif::ipi[]
+ifdef::upi[]
+. Record the AMI id for your specific region. If you use the CloudFormation template to deploy your worker nodes, you can update the `worker0.type.properties.ImageID` parameter with this value.
```

Comment:
Just to verify, the other flavor of this is "you must" but this is "you can" - want to make sure that's as intended :)

Reply:
Not intended. Thank you. :)